backend/app/services/chat/preprocessing/contexts.py

```
@@ -444,6 +469,117 @@ def _sync_kb_contexts_to_task(
+    """
+    import asyncio
+
+    from app.services.sandbox_file_syncer import (
+        get_sandbox_file_syncer,
+        sync_attachment_to_sandbox_background,
+    )
+
+    try:
+        # Check if sandbox is healthy synchronously first (fast check)
+        # This avoids fetching binary data if sandbox is not running
+        syncer = get_sandbox_file_syncer()
```

Ruff (0.14.14)

[error] Local variable `syncer` is assigned to but never used

Remove assignment to unused variable `syncer`

(F841)